src/UITests/UILibrary/LiveMode.cs

```diff
@@ -1,17 +1,46 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.SharedUx.Properties;
 using OpenQA.Selenium.Appium.Windows;
+using UITests.Utilities;
 
 namespace UITests.UILibrary
 {
     public class LiveMode
     {
         WindowsDriver<WindowsElement> Session;
 
+        const string OpenFileToolBarAutomationID = "1001";
+        const string OpenFileFolderTextBoxAutomationID = "41477";
+        const string OpenFileFileTextBoxAutomationID = "1148";
+
         public LiveMode(WindowsDriver<WindowsElement> session)
         {
             Session = session;
         }
-        public bool OpenFile(string folder, string fileName) => false;
+
+        public bool OpenFile(string folder, string fileName)
+        {
+            if (!Session.FindAndClickByAutomationID(AutomationIDs.MainWinLoadButton)) return false;
+
+            if (!Session.FindAndClickByAutomationID(OpenFileToolBarAutomationID)) return false;
+
+            try
+            {
+                var folderTextbox = Session.FindElementByAccessibilityId(OpenFileFolderTextBoxAutomationID);
+                folderTextbox.SendKeys(folder);
+                folderTextbox.SendKeys(OpenQA.Selenium.Keys.Enter);
+
+                var fileTextbox = Session.FindElementByAccessibilityId(OpenFileFileTextBoxAutomationID);
+                fileTextbox.SendKeys(fileName);
+                fileTextbox.SendKeys(OpenQA.Selenium.Keys.Enter);
+
+                return true;
+            }
+            catch
+            {
+                return false;
```

Comment:
>false [](start = 23, length = 5)

I have mixed feelings on returning bool. Would the code be simpler if we simply let the Exception leak through to the unit test? The unit test would still fail, but without any need of error handling in the unit test itself

Reply:
My thinking with the bools is they gives us control over whether or not we enforce that a navigation event succeeds. The primary scenario I had in mind was the telemetry dialog--we want to dismiss it if it's there, but we don't care if it isn't (when we aren't testing it). With this syntax, you can do `Assert.IsTrue(<navigation>)` if you want to require the event to succeed, and just call it without Assert if the navigation is not really part of the test. 

That being said, I had that idea before we finalized our plan to control the user config in these tests. I can no longer think of a scenario where a navigation event doesn't need to happen successfully, so I think you're right that these don't give us a benefit.